home.html

```diff
@@ -169,75 +175,9 @@ <h2>Join our newsletter</h2>
             </div>
         </div>
         <div class="footer-bar">
-            Copyright <i class="ri-copyright-line"></i> 2024 Bindawdi Enterprise. All rights reserved.
+            Copyright <i class="ri-copyright-line"></i> 2024 Bindawdi Enterprise.
```

Comment:
It seems the green background-color is over powering the white color text. Try tweaking it a bit to be very visible.